File: Juan_Carlos_Caballero/Practicas python/Persona.py
class Persona:
    def __init__(self, nombre, edad, color, peso, estatura):
        self.nombre = nombre
        self.edad = edad
        self.color = color
        self.peso = peso
        self.estatura = estatura
    
    def get_nombre(self):
        return self.nombre
    
    def set_nombre(self, nombre):
        self.nombre = nombre
    
    def get_edad(self):
        return self.edad
    
    def set_edad(self, edad):
        self.edad = edad
    
    def get_color(self):
        return self.color
    
    def set_color(self, color):
        self.color = color
    
    def get_peso(self):
        return self.peso
    
    def set_peso(self, peso):
        self.peso = peso
    
    def get_estatura(self):
        return self.estatura
    
    def set_estatura(self, estatura):
        self.estatura = estatura
    
    def __str__(self):
        return f"Persona{{nombre='{self.nombre}', edad={self.edad}, color='{self.color}', peso={self.peso}, estatura={self.estatura}}}"

File: Francisco_Torres/Practicas De Java/practica2/src/main/java/com/poo/sis401s/Persona.java
package com.poo.sis401s;

public class Persona {
    private String nombre;
    private int edad;
    private String color;
    private int peso;
    private double estatura;

    public Persona(String nombre, int edad, String color, int peso, double estatura) {
        this.nombre = nombre;
        this.edad = edad;
        this.color = color;
        this.peso = peso;
        this.estatura = estatura;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public int getPeso() {
        return peso;
    }

    public void setPeso(int peso) {
        this.peso = peso;
    }

    public double getEstatura() {
        return estatura;
    }

    public void setEstatura(double estatura) {
        this.estatura = estatura;
    }

    @Override
    public String toString() {
        return nombre + " | Edad: " + edad + " | Color: " + color +
               " | Peso: " + peso + "kg | Estatura: " + estatura + "m";
    }
}